docs/assets/js/examples.js

```diff
@@ -0,0 +1,263 @@
+    })
+  })
+
+  // Open example and scroll to examples section when user clicks on
+  // tech in the header
+  $('.link-to-test-with-terratest').on('click', function() {
+    openExample('index_page', $(this).data('target'))
+    scrollToTests()
+  })
+
+  // Switch between code snippets (files)
+  $('.examples__tabs .tab').on('click', function() {
+    $(this).parents('.examples__tabs').find('.tab').removeClass('active')
+    $(this).addClass('active')
+
+    $(this).parents('.examples__block').find('.examples__code').removeClass('active')
+    $($(this).data('target')).addClass('active')
+
+    loadCodeSnippet()
+  })
+
+  // Open dropdown of technologies to select
+  $('.navs__dropdown-arrow').on('click', function() {
+    $('.navs__dropdown-menu').toggleClass('active')
+  })
+
+  // Open popup when user click on circle with the number
+  $('.examples__container').on('click', '.code-popup-handler', function() {
+    const isActive = $(this).hasClass('active')
+    $('.code-popup-handler').removeClass('active')
+    if (!isActive) {
+      $(this).addClass('active')
+    }
+  })
+
+  function scrollToTests() {
+    $([document.documentElement, document.body]).animate({
+        scrollTop: $('#index-page__test-with-terratest').offset().top
+    }, 500)
+  }
+
+  function openExample(exampleContainerId, target) {
+    // Change active nav in window state and rebuild navigation first
+    const $ecId = $('#'+exampleContainerId)
+    window.examples.nav[exampleContainerId].current = target
+    buildExamplesNav()
+
+    // Change active tab in navigation
+    $ecId.find('.examples__nav-item').removeClass('active')
+    const jTarget = $('.navs .examples__nav-item[data-id="'+target+'"]')
+    jTarget.addClass('active')
+
+    // Change the block below navigation (with code snippets)
+    $ecId.find('.examples__block').removeClass('active')
+    $ecId.find('#example__block-' + target).addClass('active')
+
+    // Set current tab
+    $ecId.find('.examples__nav .navs').removeClass('active')
+
+
+    loadCodeSnippet()
+  }
+
+  function loadCodeSnippet() {
+    $('.examples__block.active .examples__code.active').each(async function (i, activeCodeSnippet) {
+      const $activeCodeSnippet = $(activeCodeSnippet)
+      const exampleTarget = $(this).data('example')
+      const fileId = $(this).data('target')
+      if (!$activeCodeSnippet.data('loaded')) {
+        const response = await fetch($activeCodeSnippet.data('url'))
+        const json = await response.json()
+        $activeCodeSnippet.attr('data-loaded', true)
+        const content = atob(json.content)
+        findTags(content, exampleTarget, fileId)
+        $activeCodeSnippet.find('code').html(content)
+        Prism.highlightAll()
+      }
+      updatePopups()
+      openPopup(exampleTarget, 1)
+    })
+  }
+
+  function findTags(content, exampleTarget, fileId) {
+    let tags = []
+    // let regexpTags =  /data "(\w+)" "(\w+)" {/mg
+    let regexpTags = /website::tag::(\d)::(.*)/mg
```

Comment:
Strip any whitespace before the content:

```suggestion
    let regexpTags = /website::tag::(\d)::(\s*)(.*)/mg
```

That way, we can put a space before the comment to make it slightly more legible:

```go
// website::tag::1:: The space to the left makes this comment slightly more legible
someCode()
```